lib/syscall_shim/Config.uk

```diff
@@ -39,4 +39,11 @@ if LIBSYSCALL_SHIM
 		bool "Enable debug messages"
 		default n
 		select LIBUKDEBUG_PRINTD
+
+	config LIBSYSCALL_SHIM_LEGACY_VERBOSE
+		bool "Warn for unimplemented legacy syscalls"
```

Comment:
Better to call it unmapped, removed, suppressed or similar instead of unimplemented - this might be confusing.